spec/lib/miq_automation_engine/models/miq_ae_domain_spec.rb

```diff
@@ -316,6 +324,37 @@
       expect(dom1.attributes).to have_attributes(commit_hash)
     end
 
+    it "import a domain given a git url and a branch" do
+      options = {'url' => url, 'tenant_id' => @user.current_tenant.id,
+                 'ref' => branch_name, 'ref_type' => 'BrancH',
+                 'userid' => "fred", "password" => "secret" }
```

Comment:
Can you put one option per line and align the hash rockets?  As is, this is very hard to read and see what's actually going on.  For example, only a keen eye would see that you put ref_type as "BrancH", but that's very important for the test.  Keeping it one per line might let that pop out more to a casual reader.

Incidentally, you might want to put a comment that that "BrancH" is intentional in order to test strange inputs.